src/platforms/javascript/guides/gatsby/index.mdx

```diff
@@ -24,47 +24,86 @@ yarn add @sentry/gatsby
 
 ## Connecting the SDK to Sentry
 
-Register the plugin in your Gatsby configuration file (typically `gatsby-config.js`).
+You can configure the SDK in one of two ways discussed below - creating a configuration file, or defining the options along with the Gatsby configuration. If you define options in both places, the SDK will prioritize the init in the configuration file and ignore the options in the Gatsby configuration.
+
+### Sentry Configuration File
+
+<Note>
+
+The minimum version supporting the Sentry configuration file is `6.14.0`.
+
+</Note>
+
+Using a Sentry configuration file is the approach we recommend since it supports defining non-serializable options in the `init`. Note that you still need to include the plugin, even if you don't set any options.
 
 ```javascript {filename:gatsby-config.js}
 module.exports = {
-  // ...
   plugins: [
     {
       resolve: "@sentry/gatsby",
-      options: {
-        dsn: "___PUBLIC_DSN___",
-      },
     },
-    // ...
-  ],
+  ]
 };
 ```
 
-## Options
+And configure your `Sentry.init`:
```

Comment:
This doesn't flow because the last bit of text isn't written as a direction/command. So I'd just remove the "and".

```suggestion
Configure your `Sentry.init`:
```